middleware/jwt/jwt.go

```diff
@@ -20,13 +21,35 @@ import (
 	"github.com/SermoDigital/jose/jws"
 	"github.com/SermoDigital/jose/jwt"
 	"github.com/mendsley/gojwk"
+	yaml "gopkg.in/yaml.v2"
```

Comment:
Something tells me you don't need the `yaml` rename here, can you check that please?

Reply:
Yup, I'll fix it. `goimports` did this for some reason.